src/couchapps/ReqMgr/views/bydate/map.js

```diff
@@ -6,7 +6,7 @@ function(doc)
 	if (doc.RequestDate && (doc.RequestDate.length > 0)){
       var dateArray = doc.RequestDate;
       // get year, month, day, min
```

Comment:
it's actually year, month, day and hour.
Can you also properly indent the if above (and these lines) such that it's more readable?